packages/astro/package.json

```diff
@@ -29,6 +29,8 @@
     "./client": "./client.d.ts",
     "./client-base": "./client-base.d.ts",
     "./astro-jsx": "./astro-jsx.d.ts",
+    "./tsconfigs/*.json": "./tsconfigs/*",
+    "./tsconfigs/*": "./tsconfigs/*.json",
```

Comment:
We need those two because in a `tsconfig.json`, you can optionally omit the file extension